src/components/FilteredProducts.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Pagination from './Pagination'
+
+const FilteredProducts = (params) => {
```

Comment:
I recommend that the props or params to be destructuted ie
```suggestion
const FilteredProducts = ({data}) => {
```
This way you know exactly what you are getting by just opening the component and don't have to be searching for params.something
Do this as much as possible in all further components